.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+dist: xenial
+node_js:
+  - "lts/*"
+# notifications:
+#   email:
+#     recipients:
+#       - amp-build-cop@grotations.appspotmail.com
+#     on_success: change
+#     on_failure: change
```

Comment:
These are notifications for the build cop rotation, so might as well remove.

Reply:
Done